File: src/main/java/eu/mshade/enderframe/item/metadata/HideFlagsItemStackMetadata.java
package eu.mshade.enderframe.item.metadata;

import eu.mshade.enderframe.item.ItemFlag;
import eu.mshade.enderframe.metadata.MetadataKeyValue;
import eu.mshade.enderframe.metadata.itemstack.ItemStackMetadataKey;

import java.util.List;
import java.util.Set;

public class HideFlagsItemStackMetadata extends MetadataKeyValue<Set<ItemFlag>> {

    public HideFlagsItemStackMetadata(Set<ItemFlag> metadataValue) {
        super(ItemStackMetadataKey.HIDE_FLAGS, metadataValue);
    }

}

File: src/main/java/eu/mshade/enderframe/item/metadata/HideFlagsItemStackMetadata.kt
package eu.mshade.enderframe.item.metadata

import eu.mshade.enderframe.item.ItemFlag
import eu.mshade.enderframe.item.ItemStackMetadataKey
import eu.mshade.enderframe.metadata.MetadataKeyValue

class HideFlagsItemStackMetadata(metadataValue: Set<ItemFlag>) :
    MetadataKeyValue<Set<ItemFlag>>(ItemStackMetadataKey.HIDE_FLAGS, metadataValue)
